src/Tribe/Views/V2/Views/Widgets/List_Widget_View.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * The List Widget View.
+ *
+ * @package Tribe\Events\Views\V2\Views\Widgets
+ * @since TBD
+ */
+
+namespace Tribe\Events\Views\V2\Views\Widgets;
+
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\Views\Traits\List_Behavior;
+use Tribe__Context;
+use Tribe__Utils__Array as Arr;
+
+/**
+ * Class List_Widget_View
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Views\Widgets
+ */
+class List_Widget_View extends View {
+	use List_Behavior;
+	/**
+	 * Slug for this view
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $slug = 'list-widget';
+
+	/**
+	 * Visibility for this view.
+	 *
+	 * @since TBD
+	 *
+	 * @var bool
+	 */
+	protected static $publicly_visible = true;
+
+	/**
+	 * {@inheritDoc}
```

Comment:
:no_entry_sign: Doc comment for parameter "$context" missing

<sub>hash: cddb382415b181624c550319b41a9bd5</sub>